devices.js

```diff
@@ -9940,8 +9940,8 @@ const devices = [
         toZigbee: [tz.on_off],
         meta: {configureKey: 1, multiEndpoint: true},
         configure: async (device, coordinatorEndpoint) => {
-            await bind(device.getEndpoint(1), coordinatorEndpoint, ['genOnOff']);
-            await bind(device.getEndpoint(2), coordinatorEndpoint, ['genOnOff']);
+            await bind(device.getEndpoint(3), coordinatorEndpoint, ['genOnOff']);
+            await bind(device.getEndpoint(4), coordinatorEndpoint, ['genOnOff']);
         },
         endpoint: (device) => {
             return {'l1': 1, 'l2': 2};
```

Comment:
Endpoint 3/4 don't match with 1/2 for controlling here, Does controlling them from zigbee2mqtt work correctly? Can you also share the `data/database.db` entry of this device?

Reply:
I agree that endpoints don't match and it was unusual for me too, but after changing endpoints to 3 and 4 it works correctly. Here's the `database.db`
`{
 "id": 8,
 "type": "Router",
 "ieeeAddr": "0x680ae2fffef6ca76",
 "nwkAddr": 17159,
 "manufId": 4633,
 "manufName": "ORVIBO",
 "powerSource": "Unknown",
 "modelId": "9ea4d5d8778d4f7089ac06a3969e784b",
 "epList": [1, 2, 242, 3, 4],
 "endpoints": {
  "1": {
   "profId": 260,
   "epId": 1,
   "devId": 256,
   "inClusterList": [0, 3, 4, 5, 6],
   "outClusterList": [25],
   "clusters": {
    "genBasic": {
     "attributes": {
      "modelId": "9ea4d5d8778d4f7089ac06a3969e784b",
      "manufacturerName": "ORVIBO",
      "powerSource": 0,
      "zclVersion": 3,
      "dateCode": "20191120",
      "swBuildId": "0.1.1"
     }
    },
    "genOnOff": {
     "attributes": {
      "onOff": 0
     }
    }
   },
   "binds": []
  },
  "2": {
   "profId": 260,
   "epId": 2,
   "devId": 256,
   "inClusterList": [0, 3, 4, 5, 6],
   "outClusterList": [25],
   "clusters": {
    "genOnOff": {
     "attributes": {
      "onOff": 0
     }
    },
    "genBasic": {
     "attributes": {}
    }
   },
   "binds": []
  },
  "3": {
   "profId": 260,
   "epId": 3,
   "devId": 256,
   "inClusterList": [0, 3, 4, 5, 6],
   "outClusterList": [25],
   "clusters": {
    "genBasic": {
     "attributes": {}
    }
   },
   "binds": [{
    "cluster": 6,
    "type": "endpoint",
    "deviceIeeeAddress": "0x00124b0018e20942",
    "endpointID": 1
   }]
  },
  "4": {
   "profId": 260,
   "epId": 4,
   "devId": 0,
   "inClusterList": [0, 3, 23],
   "outClusterList": [3, 6],
   "clusters": {
    "genBasic": {
     "attributes": {}
    }
   },
   "binds": [{
    "cluster": 6,
    "type": "endpoint",
    "deviceIeeeAddress": "0x00124b0018e20942",
    "endpointID": 1
   }]
  },
  "242": {
   "profId": 41440,
   "epId": 242,
   "devId": 97,
   "inClusterList": [],
   "outClusterList": [33],
   "clusters": {},
   "binds": []
  }
 },
 "dateCode": "20191120",
 "swBuildId": "0.1.1",
 "zclVersion": 3,
 "interviewCompleted": true,
 "meta": {
  "configured": 1
 },
 "lastSeen": 1593783416848
}`

Reply:
It indeed also has endpoint 3/4, to be on the safe side, can we also leave 1/2 there? So configure will looks like this:

```js
            await bind(device.getEndpoint(1), coordinatorEndpoint, ['genOnOff']);
            await bind(device.getEndpoint(2), coordinatorEndpoint, ['genOnOff']);
            await bind(device.getEndpoint(3), coordinatorEndpoint, ['genOnOff']);
            await bind(device.getEndpoint(4), coordinatorEndpoint, ['genOnOff']);
```

Reply:
Yes, we can leave it that way.
I've just checked and it's working correctly
`zigbee2mqtt:info  2020-07-10 15:59:40: MQTT publish: topic 'zigbee2mqtt/inwall1', payload '{"state_l2":"OFF","linkquality":105,"state_l1":"ON"}'
zigbee2mqtt:info  2020-07-10 15:59:41: MQTT publish: topic 'zigbee2mqtt/inwall1', payload '{"state_l2":"OFF","linkquality":99,"state_l1":"OFF"}'
zigbee2mqtt:info  2020-07-10 15:59:43: MQTT publish: topic 'zigbee2mqtt/inwall1', payload '{"state_l2":"ON","linkquality":94,"state_l1":"OFF"}'
zigbee2mqtt:info  2020-07-10 15:59:44: MQTT publish: topic 'zigbee2mqtt/inwall1', payload '{"state_l2":"OFF","linkquality":99,"state_l1":"OFF"}'`

Reply:
How to get an update about the state of switch after restarting the zigbee2mqtt? With manufacturer's hub the device reports its state without a problem.
I tried to set an `availability_timeout `but it's not getting states

Reply:
@azamat5991 the zigbee2mqtt state is persistent over restarts, so unless you change the state while zigbee2mqtt is down it should be fine. To force a read send to `zigbee2mqtt/inwall1/get` with payload `{"state": ""}`. I will update your PR and merge this. Thanks!

Reply:
Hi @Koenkk,
Is there a way to do that automatically?
Like when z2m restarts, it publishes zigbee2mqtt/../get in order to get the status?
And does it work with battery powered devices?

Reply:
@azamat5991 no that's not possible currently (you have to do it externally). Battery powered devices will probably fail because they are sleeping at that time.